### ISSUE TITLE
Program crashes when using png with alpha channel

### ISSUE THREAD

**Issue by themightyoarfish** [anishathalye/neural-style#53]
When using an srgba image like this one:
![brain2](https://cloud.githubusercontent.com/assets/11613312/22921764/3e343a1e-f29b-11e6-822a-76dc1c94e452.png)
the program crashes with
```
Traceback (most recent call last):
  File "neural_style.py", line 156, in <module>
    main()
  File "neural_style.py", line 130, in main
    checkpoint_iterations=options.checkpoint_iterations
  File "/home/ubuntu/neural-style/stylize.py", line 51, in stylize
    net, _ = vgg.net(network, image)
  File "/home/ubuntu/neural-style/vgg.py", line 39, in net
    current = _conv_layer(current, kernels, bias)
  File "/home/ubuntu/neural-style/vgg.py", line 52, in _conv_layer
    padding='SAME')
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/gen_nn_ops.py", line 394, in conv2d
    data_format=data_format, name=name)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/op_def_library.py", line 703, in apply_op
    op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 2319, in create_op
    set_shapes_for_outputs(ret)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 1711, in set_shapes_for_outputs
    shapes = shape_func(op)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/common_shapes.py", line 230, in conv2d_shape
    input_shape[3].assert_is_compatible_with(filter_shape[2])
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/tensor_shape.py", line 108, in assert_is_compatible_with
    % (self, other))
ValueError: Dimensions 4 and 3 are not compatible
```

**Comment by anishathalye** [comment]
We could update the code to throw away the alpha channel (or do something else reasonable). Pull requests appreciated.

**Comment by anishathalye** [comment]
thanks for reporting. fixed by #59
